src/test/java/api/loans/LoanAPITests.java

```diff
@@ -891,6 +892,7 @@ public void canCompleteALoanByReturningTheItem()
     returnedLoan
       .put("status", new JsonObject().put("name", "Closed"))
       .put("action", "checkedin")
+      .put("checkinServicePointId", UUID.randomUUID().toString())
```

Comment:
Given that this test is the primary example for how to check in an item (and to check that it behaves correctly) and that the service point property has been added, please can a check be added that the property is correctly stored on the loan.